p1-staking-pool/src/constants.rs

```diff
@@ -9,10 +9,13 @@ pub const GAS_FOR_FT_TRANSFER: Gas = 10_000_000_000_000;
 pub const NO_DEPOSIT: Balance = 0;
 pub const ONE_YOCTO: Balance = 1;
 
-pub const NANO:u64 = 1_000_000_000;
+/// one second in nanoseconds
+pub const SECOND: u64 = 1_000_000_000;
```

Comment:
please change this back to NANO
the concept of "SECONDS" has no relation to 1e9

Reply:
Nano = 1e-9, not 1e9.
https://en.wikipedia.org/wiki/Nano-

We can change it to GIGA, but I think second, as a unit is more intuitive here.

Reply:
how about NANOSECONDS?

Reply:
NANOSECOND is indeed a bounded unit. But it must be expressed with something. 
In this context NANOSECOND = 1 because the relative unit is the block time which is expressed nanoseconds.